README.md

```diff
@@ -366,6 +366,19 @@ If your API request is being sent by jQuery as provided by Apostrophe, you're go
 
 If your API request is sent via `fetch` or another alternative to jQuery, you'll need to set the `X-XSRF-TOKEN` HTTP header to the current value of `window.apos.csrfCookieName`. This ensures the request didn't come from a sneaky form on a third-party website.
 
+## Adding CORS config
+
+It's also possible to add a specific CORS config for Headless routes. It won't protect your routes but only make sure that requests that are comming from a browser are comming from a url you authorized.
```

Comment:
How about;

"It's also possible to add a specific CORS configuration for headless routes. Note that this does not secure your routes against the use of scripts, `curl`, etc. It only prevents well-behaved browsers like Chrome from making unwanted cross-site requests."

Reply:
Modified!